tasks/e2e-simple.sh

```diff
@@ -270,6 +259,10 @@ verify_module_scope
 # Eject...
 echo yes | npm run eject
 
+# Temporary workaround for https://github.com/facebook/create-react-app/issues/6099
+rm yarn.lock
+yarn add @babel/plugin-transform-react-jsx-source @babel/plugin-syntax-jsx @babel/plugin-transform-react-jsx @babel/plugin-transform-react-jsx-self
```

Comment:
Isn't this in master now? Maybe this branch should be rebased before merging.

Reply:
The change was added to this file but i don't think it was needed. I've removed - let's see if it passes!